portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,10 @@
 
 package com.google.sps.servlets;
 
+import java.io.*; 
```

Comment:
To be very nitpicky (this would be caught by automatic linters at Google), we would sort our import statements.